src/platform/thinking/common/thinking.ts

```diff
@@ -21,24 +31,29 @@ export type ThinkingDelta = {
 	text?: string;
 	id: string;
 	metadata?: string;
-	isEncrypted?: boolean;
 } | {
 	text?: string;
 	id?: string;
 	metadata: string;
-	isEncrypted?: boolean;
-} |
-{
+} | {
 	text: string;
 	id?: string;
 	metadata?: string;
-	isEncrypted?: boolean;
 };
 
+export type EncryptedThinkingDelta = {
```

Comment:
In responses api, we can have encrypted content and summary text at the same time, are we losing that now?